casepro/settings_common.py

```diff
@@ -56,6 +56,8 @@
 SITE_HIDE_CONTACT_FIELDS = []  # Listed fields should not be displayed
 SITE_CONTACT_DISPLAY = "name"  # Overrules SITE_HIDE_CONTACT_FIELDS Options: 'name', 'uuid' or 'urns'
 SITE_ALLOW_CASE_WITHOUT_MESSAGE = True
+MAX_MESSAGE_CHARS = 140
```

Comment:
Thus far we've prefixed CasePro specific settings with SITE_ so let's continue with that.